packages/bitcore-node/src/providers/chain-state/evm/p2p/rpcs/gethRpc.ts

```diff
@@ -44,20 +45,29 @@ export class GethRPC implements IRpc {
       params: [this.web3.utils.toHex(blockNumber), { tracer: 'callTracer' }],
       jsonrpc: '2.0',
       id: 1
+    }).catch(err => {
```

Comment:
Let's put this in a try..catch instead of doing a .catch here. For consistency, I'd like to stick to one pattern of error handling. The only time we'd use a .catch is if we aren't awaiting the promise.

Reply:
I have corrected this. Thanks for the input.